### ISSUE TITLE
Suggestion: Self-reposted quote-replies should bridge when replying to unbridged accounts (Bsky→Masto)

### ISSUE THREAD

**Issue by jmcken1** [snarfed/bridgy-fed#1381]
This takes a little explaining: By default, public posts and replies have different visibility; anyone viewing your user feed (and not the “Replies” tab) sees your public posts but not your replies, whereas posting a reply to someone else hides that post from your user feed. You can repost your reply, but that won’t necessarily make the context of your reply immediately clear.

So what I do (on Bluesky) is something I call “quote-replying”. First, I quote the post I’m replying to with my comment, then I post my quote-reply as a reply to their post, and finally I repost that quote-reply into my own feed. This way both my followers and anyone viewing the post I’m replying to can see it, and I don’t have to post the same thing twice. [This is what it looks like in practice](https://bsky.app/profile/joemcken.net/post/3l6ga5zwra32l) (content warning: politics).

Now, when I post this in reply to a bridged account (such as my own), the quote-reply shows up on my bridged Mastodon account as a comment followed by a link to the post I’m reply-quoting. All good. But when I’m replying to an unbridged account (like with the example linked above), the quote-reply doesn’t bridge over to Masto, even when I repost it on Bsky.

I understand that currently only replies to other bridged accounts will appear in your own bridged account. But perhaps an exception should exist when the user reposts their own reply, even if it’s to an unbridged account? After all, if I repost my own reply into my main feed, it’s because I want my own post to be publicly visible to my followers, just as it is on Bsky.

(I know this is all a bit convoluted; please let me know if it needs any clarification.)

**Comment by qazmlp** [comment]
As long as the quoted and replied-to post are the exact same, that seems like no context would be lost in translation 🤔

I was concerned Bluesky would embed with the human-readable web URL, but [it's resolved in the embed](https://atproto-browser.vercel.app/at/did:plc:nm5i2rwxc6vmgc7tu2gpwpev/app.bsky.feed.post/3l6ga5zwra32l) so that:

```rs
value.embed.$type == "app.bsky.embed.record" && value.embed.record == value.reply.parent
```

**Comment by jmcken1** [comment]
@snarfed Indeed, transforming the quote-reply on Bsky into a regular quote-post on Masto (meaning a post with a “RP:” link, as quote-posts are currently bridged, IIRC) would be perfectly fine.

**Comment by NUMB3RS00** [comment]
👍
